src/Symfony/Component/Security/Http/Firewall/LogoutListener.php

```diff
@@ -57,11 +57,17 @@ public function __construct(TokenStorageInterface $tokenStorage, HttpUtils $http
             throw new InvalidArgumentException('The CSRF token manager should be an instance of CsrfProviderInterface or CsrfTokenManagerInterface.');
         }
 
+        if (isset($options['intention'])) {
+            @trigger_error('The "intention" option for the '.__CLASS__.' is deprecated since version 2.8 and will be removed in 3.0. Use the "csrf_token_id" option instead.', E_USER_DEPRECATED);
+
+            $options['csrf_token_id'] = $options['intention'];
+        }
```

Comment:
mistakes happen and we need to help people fix them so if it's not too much work I'd say yes...